charts/oes/config/oes-visibility/visibility-local.yml

```diff
@@ -1,3 +1,11 @@
+message-broker:
+  enabled: false
+  username: rabbitmq
```

Comment:
Need to be picked from values.yaml file..
We should not hard code this.

Reply:
Updated the visibility-secret file to fetch username/password from values.yaml file